src/frontend/ikoner/LeggtilSirkel.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+interface ILeggtilSirkel {
+    className?: string;
+    heigth?: number;
+    width?: number;
+}
+
+const LeggtilSirkel: React.FC<ILeggtilSirkel> = ({ className, heigth, width }) => {
```

Comment:
LeggTilSirkel?

Reply:
Bedre forslag? Et ikon som er en sirkel med "+" inni 😃

Reply:
Hehe vet ikke, `LeggTilIkon` ? 
Annat spørsmål... Burde vi ha suffix på ikoner? Gjør det litt tydligere i koden att det faktiskt er en ikon som vises?

Reply:
`LeggTilIkon` synes jeg var fin. Hvis vi får flere LeggTilIkoner, så kan vi evt. endre navnet til `LeggTilSirkelIkon`?

Og ja, er enig med at det kan være nyttig å bruke suffixer. Jeg tror vi har en del ikoner som bør endres til å ha suffixer hvis vi bestemmer oss for dette.
Det samme gjelder f.eks. knapper.

Reply:
Vi endret til LeggTilMedSirkel. Syntes det var ganske beskrivende.